core/src/main/resources/application.conf

```diff
@@ -351,6 +351,8 @@ services {
   }
   MetadataService {
     class = "cromwell.services.metadata.impl.MetadataServiceActor"
+    // Set this value to "Inf" to turn off metadata summary refresh.
+    metadata-summary-refresh-interval = "Inf"
```

Comment:
I do like the idea of specifying what the defaults are when they're buried in the scala code